app/controllers/downvotes_controller.rb

```diff
@@ -7,30 +7,34 @@ def create
         review_id: @review.id,
         user_id: current_user.id
       )
-      @review.score -= 1
-
-      if @downvote.save && @review.save
-        flash[:notice] = "Downvote created successfully."
-        redirect_to @beach
-      else
-        flash[:notice] = "Downvote not created successfully."
-        render :new
+      respond_to do |format|
+        if @downvote.save
+          format.html { redirect_to :back,
+            notice: "Downvote created successfully." }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Expression at 13, 54 should be on its own line.